src/components/transactions/MigrateV3/MigrateV3ModalContent.tsx

```diff
@@ -68,7 +70,34 @@ export const MigrateV3ModalContent = () => {
   if (txError && txError.blocking) {
     return <TxErrorView txError={txError} />;
   }
-  if (migrateTxState.success) return <TxSuccessView action={<Trans>Migrated</Trans>} />;
+
+  const handleRoute = () => {
+    if (currentMarket === CustomMarket.proto_polygon) {
```

Comment:
@MareskoY the fork tests didnt work on polygon / avax cause currrent market says `polygon_fork` --> i dont think it makes sense to handle this clause wdyt? cc @grothem ?

Reply:
do we usually test against this case?

![Screenshot 2023-02-13 at 19 24 18](https://user-images.githubusercontent.com/8342048/218554432-a91bbe15-cde5-4b6b-b0d0-a6dffd3c8f96.png)

Reply:
> @MareskoY the fork tests didnt work on polygon / avax cause currrent market says `polygon_fork` --> i dont think it makes sense to handle this clause wdyt? cc @grothem ?

it's not necessary functionality for testing (e2e uses another way to switch markets)

but if it is easy to fix for the forks will be good to fix